src/create_torrent.cpp

```diff
@@ -149,11 +149,11 @@ namespace libtorrent
 					&& (flags & create_torrent::symlinks))
 				{
 					std::string sym_path = get_symlink_path(f);
-					fs.add_file(l, 0, file_flags, s.mtime, sym_path);
+					fs.add_file(l, 0, file_flags, std::time_t(s.mtime), sym_path);
```

Comment:
would it make sense to change the ``mtime``, ``atime`` and ``ctime`` members to be ``std::time_t``?
The risk I can see is that some old ABIs may still use a 32 bit ``mtime``. There's still some time out though until that may become a problem.